lib/elixir/docs.exs

```diff
@@ -4,36 +4,39 @@
   groups_for_modules: [
     # [Kernel, Kernel.SpecialForms],
 
-    "Data & Behaviours": [
-      Access,
+    "Basic Types": [
       Atom,
       Base,
       Bitwise,
       Calendar,
       Calendar.ISO,
       Date,
-      Date.Range,
       DateTime,
-      Enum,
       Exception,
       Float,
       Integer,
-      Keyword,
-      List,
-      Map,
-      MapSet,
       NaiveDateTime,
-      Range,
       Record,
       Regex,
-      Stream,
       String,
       Time,
       Tuple,
```

Comment:
Should `Tuple` also live in collections?

Reply:
I would say Tuple is a composite type but not a collection.

Reply:
Cool

Reply:
The first paragraph of the `Tuple` documentation says: [`Tuples are ordered collections of elements`](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/tuple.ex#L5). Should we update the documentation expressing is a composite type?

The [Erlang documentation](http://erlang.org/doc/reference_manual/data_types.html) says:

> A tuple is a compound data type with a fixed number of terms:
>
> `{Term1,...,TermN}`
>
> Each term `Term` in the tuple is called an *element*. The number of elements is said to be the size of the tuple.